src/main/java/org/selenium/pages/DressesPage.java

```diff
@@ -0,0 +1,54 @@
+package org.selenium.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.selenium.ojb.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class DressesPage extends BasePage {
+    private By categoryDresses = By.xpath("//div[@id='block_top_menu']/ul/li[2]/a");
+    private By popup = By.xpath("//div[@id='layer_cart']");
+    private By continueShop = By.xpath("//span[@title='Continue shopping']");
+    private By containerProducts = By.xpath("//ul[@class='product_list grid row']");
+    private By products = By.xpath("//ul[@class='product_list grid row']/li");
+    private By addToCartButton = By.xpath(".//a[@title='Add to cart']");
+
+    private By productName = By.xpath("//span[@id='layer_cart_product_title']");
+    private By productAttribute = By.xpath("//span[@id='layer_cart_product_attributes']");
+    private By productPrice = By.xpath("//span[@id='layer_cart_product_price']");
+
+    public List<Product> addToCart(int count, List<Product> products) {
```

Comment:
Why do you need products as second parameter?

Reply:
Fixed it